sale_stock_picking_note/views/stock_picking_view.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<!-- Copyright 2021 Tecnativa - Víctor Martínez
+     License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl). -->
+<odoo>
+    <record id="view_picking_form" model="ir.ui.view">
+        <field name="model">stock.picking</field>
+        <field name="inherit_id" ref="stock.view_picking_form" />
+        <field name="arch" type="xml">
+            <page name="note" position="after">
+                <page string="Customer Note" name="customer_note">
+                    <field
+                        name="customer_note"
```

Comment:
A placeholder putting "Comments that will be printed in the delivery slip" or similar will help user.